.babelrc

```diff
@@ -7,7 +7,8 @@
     "transform-runtime",
     "add-module-exports",
     "transform-decorators-legacy",
-    "transform-react-display-name"
+    "transform-react-display-name",
+    "transform-class-properties"
```

Comment:
No need for this, already a part of babel-preset-stage-2, which is part of stage-0 that we use.